### DISCUSSION TITLE
我想问下是不是没有上下文记忆呢~

### DISCUSSION THREAD

**Discussion by c913168497** [unit-mesh/auto-dev#201] [Ideas]
![image](https://github.com/unit-mesh/auto-dev/assets/23418458/fb22ee64-6bc4-4563-9c3a-74769820a4af)

**Comment by phodal** [comment]
估计是改出  bug  了

**Reply by phodal** [reply]
不太对，能给一下详细的步骤吗

**Reply by c913168497** [reply]
我是通过那个 Generator SQL 自动生成SQL ，操作的呢

**Reply by phodal** [reply]
应该是之前有人报过相关的  bug，所以生成 SQL 后会清空历史

**Reply by phodal** [reply]
看逻辑好像也不会，会不会上下文超了？

**Reply by phodal** [reply]
应该是出  bug  了，我看看怎么修

**Reply by c913168497** [reply]
哈哈 我看到你更新了代码 我试试~

**Comment by phodal** [comment]
@c913168497 可以等这个  build  完，试试这个新版本：https://github.com/unit-mesh/auto-dev/actions/runs/9394457395

**Reply by c913168497** [reply]
好的~ 辛苦
